ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -24,6 +24,7 @@
 #include "stdlib.h"
 
 #include "PrjFSLib.h"
+#include "../PrjFSKext/ArrayUtilities.hpp"
```

Comment:
Minor: Now that we're using `Array_Size` in user-mode could you move its implementation from `"../PrjFSKext/ArrayUtilities.hpp"` to  `"../PrjFSKext/public/ArrayUtils.hpp"`?

As a general rule we should only be including `"../PrjFSKext/public` header files in user mode.